tests/phpunit/tests/formatting/emoji.php

```diff
@@ -122,15 +122,15 @@ public function test_wp_emoji_list_returns_data() {
 		$entities = _wp_emoji_list( 'entities' );
 		$this->assertNotEmpty( $entities, 'Entities should not be empty' );
 		$this->assertIsArray( $entities, 'Entities should be an array' );
-		// Emoji 15 contains 3718 entities, this number will only increase.
-		$this->assertGreaterThanOrEqual( 3718, count( $entities ), 'Entities should contain at least 3718 items' );
+		// Emoji 17 contains 4007 entities, this number will only increase.
+		$this->assertGreaterThanOrEqual( 4007, count( $entities ), 'Entities should contain at least 4007 items' );
 		$this->assertSame( $default, $entities, 'Entities should be returned by default' );
 
 		$partials = _wp_emoji_list( 'partials' );
 		$this->assertNotEmpty( $partials, 'Partials should not be empty' );
 		$this->assertIsArray( $partials, 'Partials should be an array' );
-		// Emoji 15 contains 1424 partials, this number will only increase.
-		$this->assertGreaterThanOrEqual( 1424, count( $partials ), 'Partials should contain at least 1424 items' );
+		// Emoji 17 contains 1438 partials, this number will only increase.
+		$this->assertGreaterThanOrEqual( 1438, count( $partials ), 'Partials should contain at least 1438 items' );
```

Comment:
```suggestion
		$this->assertEquals( 1438, count( $partials ), 'Emoji 17 contains 1,438 partials.' );
```
See suggestion above.